src/main/java/com/name/OX/FinishState.java

```diff
@@ -0,0 +1,17 @@
+package com.name.OX;
+
+/**
+ * @author Ola Podorska
+ */
+class FinishState implements GameState {
+    @Override
+    public GameState nextState() {
```

Comment:
Tu trzeba coś pomyśleć z tym zakończeniem :smile: Zwracanie nullka to takie nieeleganckie ^.^